src/core/common/notifier.cpp

```diff
@@ -332,6 +332,14 @@ const char *Notifier::FlagToString(otChangedFlags aFlag) const
         retval = "ChanMask";
         break;
 
+    case OT_CHANGED_BORDER_AGENT_STATE:
+        retval = "AgentState";
```

Comment:
"Agent" seems a big generic. Can we get "Border" or some representation in there somehow?

Reply:
Sure. Changed it to "BorderAgentState"...